include/readout/models/SourceEmulatorModel.hpp

```diff
@@ -216,6 +231,10 @@ class SourceEmulatorModel : public SourceEmulatorConcept
   std::string m_name;
   bool m_is_configured = false;
   double m_rate_khz;
+
+  std::vector<bool> m_dropouts; // Random population
+
+  static constexpr int m_dropouts_length = 10000; // Random population size
```

Comment:
Config? (With this sensible default it will never be put in app confgens anyway.) But if anyone wants something more dynamic, let it be.